src/modules/comments.js

```diff
@@ -0,0 +1,74 @@
+// involvement API
+
+import { clearInput } from './clearInput';
+import { renderComments } from './rendComments';
+
+/* eslint-disable operator-linebreak */
```

Comment:
- [ ] linters are here to help you keep a clean code base, disabling it does not help, instead try solving the error